src/helpers/clipboard/clipboard_os_state.ts

```diff
@@ -64,6 +64,8 @@ export class ClipboardOsState extends ClipboardCellsAbstractState {
       right: activeCol + numberOfCols - 1,
       bottom: activeRow + numberOfRows - 1,
     };
+    // we want grid selection to capture the selection stream
+    this.selection.getBackToDefault();
```

Comment:
wasn't the original idea to re-introduce a command such as SET_SELECTION ? I feel like the intention would be clearer. (the command could have another name to make it clear it's the grid selection)

Reply:
unfortunately, we now use the selection in the 'finalize' of find&replace - we sometimes recompute the search & move the selection after a cell evaluation, which is why I dropped the command idea. Another plan could be to drop the move selection in certain conditions (like a cell evaluation) to avoid having your selection become mental upon external cell updates but that requires a bit moree thinking with our P, I first wanted to fix the UX issues already present :)

Reply:
ok, thanks, I understand. Let's move with this one then :)